### ISSUE TITLE
Can't create environment with the specified naming convention

### ISSUE THREAD

**Issue by anantzoid** [xingdi-eric-yuan/TextWorld-Coin-Collector#1]
While trying to generate games, I get the following error:
```
Traceback (most recent call last):
  File "tw-make.py", line 11, in <module>
    import gym_textworld  # Register all textworld environments.
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 950, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 646, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 616, in _load_backward_compatible
  File "/Users/jedi/anaconda3/envs/nlp/lib/python3.6/site-packages/gym_textworld-0.0.1-py3.6.egg/gym_textworld/__init__.py", line 3, in <module>
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 950, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 646, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 616, in _load_backward_compatible
  File "/Users/jedi/anaconda3/envs/nlp/lib/python3.6/site-packages/gym_textworld-0.0.1-py3.6.egg/gym_textworld/coin_collector.py", line 32, in <module>
  File "/Users/jedi/Projects/deep/rl/gym/gym/envs/registration.py", line 164, in register
    return registry.register(id, **kwargs)
  File "/Users/jedi/Projects/deep/rl/gym/gym/envs/registration.py", line 158, in register
    self.env_specs[id] = EnvSpec(id, **kwargs)
  File "/Users/jedi/Projects/deep/rl/gym/gym/envs/registration.py", line 71, in __init__
    raise error.Error('Attempted to register malformed environment ID: {}. (Currently all IDs must be of the form {}.)'.format(id, env_id_re.pattern))
gym.error.Error: Attempted to register malformed environment ID: twcc_easy_level1_gamesize1_step50_seed0_train. (Currently all IDs must be of the form ^(?:[\w:-]+\/)?([\w:.-]+)-v(\d+)$.)
```
It seems gym has a different regex altogether. I'm wondering did the authors customized gym too?

Thanks.

**Comment by MarcCote** [comment]
Thanks for reporting this issue. This is clearly an oversight from our side when cleaning the code. PR #2 should fix it. Also, I've found some other inconsistencies between this code and TextWorld (master). So, once https://github.com/Microsoft/TextWorld/pull/53 is merged, you should update your TextWorld package.

**Comment by anantzoid** [comment]
Thanks for the quick follow-up. Really helpful.

**Comment by MarcCote** [comment]
@anantzoid please update both this repo and TextWorld, and let us knows if it works for you.

**Comment by anantzoid** [comment]
Yes, it's working now. Thanks. 
I've also made a couple of changes that fixes things for PyTorch 0.4. Kindly let me know if you want me to submit a PR for that.

**Comment by xingdi-eric-yuan** [comment]
Hi @anantzoid , yes please, we will take a look of the PR, thanks.
